Configuration/Skimming/python/Skims_PA_cff.py

```diff
@@ -12,7 +12,7 @@
 
```

Comment:
Submitting a PR now with the change to pp event content.  Btw, we shouldn't be producing mini-AOD, as far as I understood.